### PR TITLE
Update FACP.yaml

### PR DESCRIPTION
**Pull request by RathishKJ**

### CODE REVIEW

File: ontology/yaml/resources/SAFETY/entity_types/FACP.yaml [hunk 1]
```diff
@@ -33,4 +33,11 @@ FACP_FA2X:
   - FA2X
   opt_uses:
   - panel_alarm
+  
+FACP_GASDET:
```
Comment (tasodorff) [inline ontology/yaml/resources/SAFETY/entity_types/FACP.yaml:37]: shouldnt there be a fire alarm on the fire alarm control panel?

Comment (RathishKJ) [reply ontology/yaml/resources/SAFETY/entity_types/FACP.yaml]: Yes... Used an existing cannonical

FACP_GASDET:
  description: "Fire alarm panel with gas leak detection alarm"
  is_canonical: true
  implements:
  - FACP
  - GASDET

**Comment by RathishKJ** [comment]
Closing the pull request as this is modeled along side the LDS with canonical LDS_GASDET

File: ontology/yaml/resources/SAFETY/entity_types/FACP.yaml (hunk 1 above)
Comment (RathishKJ) [reply ontology/yaml/resources/SAFETY/entity_types/FACP.yaml]: Deleted this canonical since the existing canonical of LDS_GASDET is used

**Pull request closed without merging**